data/docs_markdown/Get_Started:_Sign_Up_and_Log_In_to_io.net.md

```
@@ -0,0 +1,33 @@
+# Get Started: Sign Up and Log In to io.net
+## [](https://docs.io.net/docs/loginsignup#create-account)
+If you haven't created an account yet, registration on [id.io.net](https://id.io.net) is currently available through **email address,** **X (Twitter)** , **Apple ID** , **Google** , or **Worldcoin**.
+![](https://files.readme.io/bb5ad37a3a833e1124dbf73814beceacb730e448e5adf2165f7b32282d447a09-account1.jpg)
```

markdownlint-cli2 (0.17.2)

Images should have alternate text (alt text)

(MD045, no-alt-text)